lib/web/apiserver_test.go

```diff
@@ -346,6 +347,7 @@ func newWebSuite(t *testing.T) *WebSuite {
 	fs, err := NewDebugFileSystem("../../webassets/teleport")
 	require.NoError(t, err)
 	handler, err := NewHandler(Config{
+		ClusterFeatures:                 *modules.GetModules().Features().ToProto(),
```

Comment:
Dereferencing a pointer without an explicit check could result in a panic. If we're absolutely sure the pointer will always be non-nil, then let's add a comment so reviewers are aware. Otherwise we should check first.

Reply:
added comment in https://github.com/gravitational/teleport/pull/17834/commits/d04034bae725b15352b24af73824c89ccea085e5